lib/web_ui/test/canvaskit/image_test.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+import 'common.dart';
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+void testMain() {
+  setUpCanvasKitTest();
+
+  test('toImage succeeds', () async {
+    final ui.Image image = await _createImage();
+    expect(image.runtimeType.toString(), equals('CkImage'));
+    image.dispose();
+  // TODO(hterkelsen): https://github.com/flutter/flutter/issues/109265
+  }, skip: isFirefox || isSafari);
```

Comment:
Safari should work now. I'm actually working on a separate PR to unskip all the old tests for Safari.

Reply:
done